scripts/check_frontend_test_coverage.py

```diff
@@ -0,0 +1,213 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Check for any decreasement of 100% covered files in the frontend"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import shutil
+import subprocess
+import sys
+
+import python_utils
+
+
+def change_git_branch(branch):
+    """Changes git branch.
+
+    Args:
+      branch: string. Name of the branch to be changed.
+    """
+    task = subprocess.Popen(['git', 'checkout', branch])
+
+    task.communicate()
+    task.wait()
+
+    if branch == 'develop':
+        # This line is executed because CircleCI keeps the commits from PR
+        # branch when changing branchs. Ref ->
+        # (https://discuss.circleci.com/t/changing-git-branches-causes-
+        # commits-from-one-branch-to-carry-over-to-the-next/13562)
+        reset = subprocess.Popen(['git', 'reset', '--hard', 'origin/develop'])
+
+        reset.communicate()
+        reset.wait()
+
+
+def run_frontend_tests_script():
+    """Run the frontend tests script using subprocess."""
+    task = subprocess.Popen(['python', '-m', 'scripts.run_frontend_tests'])
+
+    task.communicate()
+    task.wait()
+
+
+def create_tmp_folder():
+    """Creates a temporary folder."""
+    if not os.path.exists('./tmp'):
+        os.mkdir('./tmp')
+
+
+def delete_tmp_folder():
+    """Delete the temporary folder."""
+    if os.path.exists('./tmp'):
+        shutil.rmtree('./tmp')
+
+
+def filter_lines(line):
+    """Check if the line has file path or total lines or covered lines
+        of the test.
+
+    Args:
+        line: str. A line from lcov file.
+
+    Returns:
+        True or false.
+    """
+    return bool(line.find('SF') >= 0 or line.find('LH') >= 0 or
+                line.find('LF') >= 0)
+
+
+def get_test_file_name(test_path):
+    """Get the file name from the absolute path.
+
+    Args:
+        test_path: string. The file's absolute path.
+
+    Returns:
+        file_name(str). The file name.
+    """
+    return test_path[test_path.rindex('/') + 1:]
```

Comment:
There is the ability for this to have a null or empty string passed in, there should be be check somewhere for that.

Reply:
Done